### ISSUE TITLE
How to setup MULTITENANT_TEMPLATE_DIRS?

### ISSUE THREAD

**Issue by mickeyckm** [bernardopires/django-tenant-schemas#256]
Hi,

I like the idea that we can setup a template folder for each tenant. How can I go about doing it? There is no example on how to do it right in the settings.py folder. 

In the documentation, it says that I need can use this format `template_dir % tenant.domain_url` but how can I access tenant object in the settings.py. Do you mean somewhere else? 

Does this work for normal apps like admin/grapelli after it is applies? 

Sorry for the multiple questions, I like the library but I'm still having issue setting it up. 

Thanks,
Mickey

**Comment by fablet** [comment]
I was wondering if you got any traction on this and if so, would you be willing to share what you discovered?

**Comment by P0oky** [comment]
Assume you project is called "my_project". And you have two tenants, with schema names "tenant_1" and "tenant_2"

then this:

    MULTITENANT_TEMPLATE_DIRS = [
        'tenant_templates'
    ]

would look in these directories for templates:

    my_project/tenant_templates/tenant_1/

    my_project/tenant_templates/tenant_2/


and this:

    MULTITENANT_TEMPLATE_DIRS = [
        'tenant/%s/templates'
    ]

would look in one of these:

    my_project/tenant/tenant_1/templates/

    my_project/tenant/tenant_2/templates/

**Comment by fablet** [comment]
Thank you.  That makes it much more clear. 
I think I was having trouble getting the settings for that right because I originally didn't have the loaders set in right for the django 1.8 switch to a template dictionary.  
Once I corrected that I had to deal with deprecations, and now I can see differences the base loading between my different tenants, but not differences in my tenant index templates.  
I do not have any cache settings on in my settings.py
Any thoughts on why this would be?
